tasks/upload.py

```diff
@@ -678,6 +686,61 @@ def _schedule_bundle_analysis_processing_task(
         )
         return res
 
+    def _schedule_test_results_processing_task(
+        self,
+        commit,
+        commit_yaml,
+        argument_list,
+        commit_report,
+        checkpoints=None,
+    ):
+        processor_task_group = []
+        for i in range(0, len(argument_list), CHUNK_SIZE):
+            chunk = argument_list[i : i + CHUNK_SIZE]
+            if chunk:
+                sig = test_results_processor_task.signature(
+                    args=(),
+                    kwargs=dict(
+                        repoid=commit.repoid,
+                        commitid=commit.commitid,
+                        commit_yaml=commit_yaml,
+                        arguments_list=chunk,
+                        report_code=commit_report.code,
+                    ),
+                )
+                processor_task_group.append(sig)
+        if processor_task_group:
+            checkpoint_data = None
+            if checkpoints:
+                checkpoints.log(UploadFlow.INITIAL_PROCESSING_COMPLETE)
```

Comment:
i think we need to use different flows for coverage, bundles, and test results. we have totally different expectations/norms for how long each takes, and putting them together in one pool makes the metrics useless